libavfilter/vf_dnn_processing.c

```diff
@@ -53,7 +53,7 @@ static const AVOption dnn_processing_options[] = {
 #if (CONFIG_LIBOPENVINO == 1)
     { "openvino",    "openvino backend flag",      0,                        AV_OPT_TYPE_CONST,     { .i64 = 2 },    0, 0, FLAGS, "backend" },
 #endif
-    DNN_COMMON_OPTIONS
+    DNN_COMMON_OPTIONS,
```

Comment:
Let's avoid formatting changes to common modules, will make it harder to rebase with upstream later on.

Reply:
Fixed